benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField-sandl-nonprod.json

```diff
@@ -23,5 +23,6 @@
   {"LiveFrom": "06/07/2022", "CaseTypeID": "Benefit", "CaseFieldID": "amendReasons", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
   {"LiveFrom": "06/07/2022", "CaseTypeID": "Benefit", "CaseFieldID": "amendReasons", "UserRole": "caseworker-sscs-judge", "CRUD": "R"},
   {"LiveFrom": "06/07/2022", "CaseTypeID": "Benefit", "CaseFieldID": "amendReasons", "UserRole": "caseworker-sscs-registrar", "CRUD": "R"},
-  {"LiveFrom": "06/07/2022", "CaseTypeID": "Benefit", "CaseFieldID": "amendReasons", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"}
+  {"LiveFrom": "06/07/2022", "CaseTypeID": "Benefit", "CaseFieldID": "amendReasons", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
+  {"LiveFrom": "12/08/2022", "CaseTypeID": "Benefit", "CaseFieldID": "hearingDateIssued", "UserRole": "caseworker-sscs-dwpresponsewriter", "CRUD": "CRUD"}
```

Comment:
If you want to add authorisations to a fixed list, you need to add it to the field that is using the fixed list not the ListElementCode. The field you are referencing is DwpState for this which already has the dwpresponsewriter Authorisations

Reply:
Don't add a field to both non-prod and prod sheets, it will cause an error. If you want something behind a feature flag and not to go to production, add it to a nonprod sheet.